fuse/ipns/ipns_unix.go

```diff
@@ -55,7 +55,7 @@ func NewFileSystem(ipfs *core.IpfsNode, sk ci.PrivKey, ipfspath, ipnspath string
 
 // Root constructs the Root of the filesystem, a Root object.
 func (f *FileSystem) Root() (fs.Node, error) {
-	log.Debug("Filesystem, get root")
+	log.Debug("gilesystem, get root")
```

Comment:
Typo ­– should be `filesystem`.

(Your editor may have a 'convert to lowercase' command that avoids this.)